functions/equals-text/1.0/index.js

```diff
@@ -0,0 +1,41 @@
+const equalsText = ({ leftValue, rightValue, comparator, toLower }) => {
+  console.log("🚀 ~ file: index.js:2 ~ equalsText ~ comparator", comparator)
```

Comment:
Remove log, not really mandatory to log this for every exectution of the function right?